src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/NativeLayoutVertexNode.cs

```diff
@@ -120,7 +120,7 @@ public override Vertex WriteVertex(NodeFactory factory)
             }
 
             Vertex signature = GetNativeWriter(factory).GetMethodSignature((uint)flags, 0, containingType, methodNameAndSig, args);
-            return factory.MetadataManager.NativeLayoutInfo.LdTokenInfoSection.Place(signature);
+            return SetSavedVertex(factory.MetadataManager.NativeLayoutInfo.LdTokenInfoSection.Place(signature));
```

Comment:
This looks reasonable, but could you write up the reasons why one would use SetSavedVertex and why one wouldn't. I see the .Place function used in both cases where we save and where we don't, and I'd like to have a clear document explaining why. (checked in as a comment or something would be adequate)